src/widgets/modals/index.js

```diff
@@ -4,7 +4,7 @@
  */
 
 export { ByProgramModal } from './ByProgramModal';
-export { ConfirmModal } from './ConfirmModal';
+export { NewConfirmModal } from '../modalChildren';
```

Comment:
We also need to rename `NewConfirmModal` to `ConfirmModal`

Reply:
@joshxg 
I would change it to ConfirmForm as Mark suggested previously if it's OK with you?

Reply:
I cannot see a suggestion for `ConfirmForm` anywhere. I think the easiest is to just use `ConfirmModal`

Reply:
@joshxg 
<img width="818" alt="Screen Shot 2020-02-17 at 2 12 44 PM" src="https://user-images.githubusercontent.com/57238075/74616977-ae9e3b00-518f-11ea-9213-163ec0226b98.png">

Do you see this comments as I do?

Reply:
@joshxg I am fine with either, so just let me know which one to use ?

Reply:
@Rebecca367 Sorry! missed that comment from @Chris-Petty -- I too am fine - you choose!

Reply:
@joshxg 
I would change NewConfirmModal to ConfirmForm as suggested and delete confirmModal after works are done if it's fine for you? What props would you recommend me using in FinalizeModal please?
![Screen Shot 2020-02-17 at 2 40 04 PM](https://user-images.githubusercontent.com/57238075/74617709-6f71e900-5193-11ea-92dd-0153f44ecb0b.png)